src/omotes_simulator_core/entities/assets/esdl_asset_object.py

```diff
@@ -90,6 +90,16 @@ def get_port_type(self, port_type: str) -> Type[esdl.Port]:
         else:
             raise ValueError(f"Port type not recognized: {port_type}")
 
+    def get_marginal_costs(self) -> float:
+        """Get the marginal costs of the asset."""
+        if self.esdl_asset.costInformation is None:
+            logger.warning(f"No cost information found for asset: {self.esdl_asset.name}")
```

Comment:
Is the warning message clear for the user? Why is the 'cost information' relevant when I want to use the marginal cost to set the priorities? 

Maybe we should mention that we here use the cost as a proxy for priority?

Reply:
How is the zero value handled in the priority?

Reply:
Zero would get highest priority

Reply:
Costinformation is the group in which the costs are set.  If you have no cost information then this does not exists. If you have not set marginal costs but other costs this attribute exists, but marginal cost not.